keyboards/lazydesigners/the30/readme.md

```diff
@@ -0,0 +1,17 @@
+# THE30
+
+![the30](https://i.loli.net/2019/11/06/KTn3dNjreigovRz.jpg)
+
+A 30% custom keyboard designed and produced by [LazyDesigners](http://lazydesigners.cn).
+
+Keyboard Maintainer: [Erovia](https://github.com/Erovia)  
+Hardware Supported: THE30  
+Hardware Availability: Check for GBs on [Geekhack](https://geekhack.org) and on [LazyDesigner's homepage.](http://lazydesigners.cn)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make lazydesigners/the30:default:dfu
+
+To enter the bootloader, either press the RESET button on the PCB or the RESET button on the FN layer.
```

Comment:
```suggestion
```

Reply:
Why remove this info?
I'd rather not let the users wonder around the net, looking for how to enter bootloader mode.

Reply:
This only really applies to the default keymap (at least, the last part anyway). It might be better placed in that readme instead.